api/web3server_utils.go

```diff
@@ -91,7 +91,7 @@ func ioAddrToEthAddr(ioAddr string) (string, error) {
 	if len(ioAddr) == 0 {
 		return "0x0000000000000000000000000000000000000000", nil
 	}
-	addr, err := util.IoAddrToEvmAddr(ioAddr)
+	addr, err := ioctl.IoAddrToEvmAddr(ioAddr)
```

Comment:
duplicate this `IoAddrToEvmAddr` code here, so we can remove dependency to 
`"github.com/iotexproject/iotex-core/ioctl/util"`

Reply:
Done, let me know what you think regarding my import of 
"github.com/iotexproject/iotex-core/ioctl/output"